e2e/playwright-ci.config.js

```diff
@@ -4,6 +4,8 @@
 
 // eslint-disable-next-line no-unused-vars
 const { devices } = require('@playwright/test');
+const MAX_FAILURES = 5;
```

Comment:
really this should be calculated from the number of retries but 5 works perfect in a sharded environment with 3 retries. We should revisit this when we refactor GHA to use shards